src/component/utils/getElementForBlockType.js

```diff
@@ -12,32 +12,23 @@
 
 'use strict';
 
+const DefaultDraftBlockRenderMap = require('DefaultDraftBlockRenderMap');
+
 import type {DraftBlockType} from 'DraftBlockType';
+import type {DraftBlockRenderMap} from 'DraftBlockRenderMap';
+
+function getElementForBlockType(
+  blockType: DraftBlockType,
+  draftBlockRenderMap: DraftBlockRenderMap
+): string {
+  const matchedBlockType = draftBlockRenderMap.get(blockType) ||
+    draftBlockRenderMap.get('unstyled');
+
+  const matchedBlockTypeTag = matchedBlockType ?
+    matchedBlockType.get('element') :
+    DefaultDraftBlockRenderMap.get('unstyled').get('element');
 
-function getElementForBlockType(blockType: DraftBlockType): string {
-  switch (blockType) {
-    case 'header-one':
-      return 'h1';
-    case 'header-two':
-      return 'h2';
-    case 'header-three':
-      return 'h3';
-    case 'header-four':
-      return 'h4';
-    case 'header-five':
-      return 'h5';
-    case 'header-six':
-      return 'h6';
-    case 'unordered-list-item':
-    case 'ordered-list-item':
-      return 'li';
-    case 'blockquote':
-      return 'blockquote';
-    case 'media':
-      return 'figure';
-    default:
-      return 'div';
-  }
+  return matchedBlockTypeTag;
 }
 
 module.exports = getElementForBlockType;
```

Comment:
Can this whole function be eliminated, with the logic above simply done inline within `DraftEditorContents`? Same for `getWrapperTemplateForBlockType`.

I don't recall whether there are other callsites for those functions, but I'm all in favor of cleaning up modules we don't need. :)